src/unique_user_email/apps.py

```diff
@@ -22,5 +22,4 @@ def ready(self):
             ),
         ]
         User._meta.constraints = User.Meta.constraints
-        # ... as long as original_attrs is not updated.
-        # User._meta.original_attrs["constraints"] = User.Meta.constraints
+        User._meta.original_attrs["constraints"] = User.Meta.constraints
```

Comment:
This will cause the auto-detector to create an `auth` migration when `unique_user_email` is installed but the migration is not run. 

I think we should have tests for both that case **and** the issue here (which is that an `auth` migration is created **after** the `unique_user_email` migration is run).

Reply:
Ah ha! It' looks like it doesn't 🕺

```
% ./manage.py showmigrations
admin
 [ ] 0001_initial
 [ ] 0002_logentry_remove_auto_add
 [ ] 0003_logentry_add_action_flag_choices
auth
 [ ] 0001_initial
 [ ] 0002_alter_permission_name_max_length
 [ ] 0003_alter_user_email_max_length
 [ ] 0004_alter_user_username_opts
 [ ] 0005_alter_user_last_login_null
 [ ] 0006_require_contenttypes_0002
 [ ] 0007_alter_validators_add_error_messages
 [ ] 0008_alter_user_username_max_length
 [ ] 0009_alter_user_last_name_max_length
 [ ] 0010_alter_group_name_max_length
 [ ] 0011_update_proxy_permissions
 [ ] 0012_alter_user_first_name_max_length
contenttypes
 [ ] 0001_initial
 [ ] 0002_remove_content_type_name
sessions
 [ ] 0001_initial
unique_user_email
 [ ] 0001_initial
% ./manage.py makemigrations auth
No changes detected in app 'auth'
```

OK, let me have a play over the weekend, and I'll get this out. 

Thanks @trottomv 🎁

Reply:
for completenes:

```
(.venv) ➜  issue5 python manage.py showmigrations
admin
 [X] 0001_initial
 [X] 0002_logentry_remove_auto_add
 [X] 0003_logentry_add_action_flag_choices
auth
 [X] 0001_initial
 [X] 0002_alter_permission_name_max_length
 [X] 0003_alter_user_email_max_length
 [X] 0004_alter_user_username_opts
 [X] 0005_alter_user_last_login_null
 [X] 0006_require_contenttypes_0002
 [X] 0007_alter_validators_add_error_messages
 [X] 0008_alter_user_username_max_length
 [X] 0009_alter_user_last_name_max_length
 [X] 0010_alter_group_name_max_length
 [X] 0011_update_proxy_permissions
 [X] 0012_alter_user_first_name_max_length
contenttypes
 [X] 0001_initial
 [X] 0002_remove_content_type_name
sessions
 [X] 0001_initial
unique_user_email
 [ ] 0001_initial
(.venv) ➜  issue5 python manage.py makemigrations
No changes detected
```